webpack.config.dev.js

```diff
@@ -12,15 +12,12 @@ const insertHotReload = (host, port, entries) => (
   R.map(R.compose(R.flatten, v => [v].concat(hotEntry(host, port))), entries)
 );
 
-const devConfig = {
+const devConfig = Object.assign(config, {
```

Comment:
We should only do `x = Object.assign({}, ...)` or `Object.assign(x, ...)`, `y = Object.assign(x, ...)` is confusing because `x` is assigned to directly. This will set `devConfig` equal to `config` here. Also `devConfig.module.loaders` below will be unaffected because this is a shallow clone.

In any case I think it's fine here to just modify `config`, nothing else is going to be using it.